Developing a new version

UladBohdan opened the pull request.

mousebaiker committed:
Refactor for automatic country processing.

This allows for automatic common and unique country extraction, easy country counting
and no manual HTML editing.

UladBohdan committed:
Merge pull request #1 from mousebaiker/master

Refactor for automatic country processing.

mousebaiker reviewed (changes requested):
This is not stable enough to be merged to master yet. Flags are not rendered properly for some countries. Language support is absent.

mousebaiker committed:
Add full country name to ISO code conversion.

This feature allows to store country data in a human-readable way and then convert it to
easier to use country codes.

Flags work now, expectw for provinces, which need to be handled separatly.

mousebaiker committed:
Display flags for provinces
